### PR TITLE
Front rating updated

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: client/src/components/comment-form/comment-form.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const CommentForm = ({productId, setComments, storeService, comments, rate}) =>
           onSubmit={addComment}
     >
       <h3> Leave a comment </h3>
-      <div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId}/></div>
+      <div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId} userId={userId} readonly={false} color="black" isSelectable={true}/></div>
```
Comment (oleksandrhodzevych) [inline client/src/components/comment-form/comment-form.js]: move to next row

Comment (oleksandrhodzevych) [inline client/src/components/comment-form/comment-form.js]: color="black" - hard code

File: client/src/components/product-details/Product-details.js [hunk 2]
```diff
@@ -158,7 +158,7 @@ const ProductDetails = ({
           />
         </Col>
         <Col className="text" id="text">
-          <StarsRating rating={product.rate} id={product.id} />
+          <StarsRating rating={product.rate} color="black" isSelectable={false} />
```
Comment (oleksandrhodzevych) [inline client/src/components/product-details/Product-details.js]: color="black" - hard code

File: client/src/components/product-list-item/Product-list-item.js [hunk 3]
```diff
@@ -21,9 +19,7 @@ function ProductListItem({ title, id, images, price, mrsp, currency, currencyIco
             <div className="productCard" id="productCard"><img id="productImg" alt={`${images}`} src={`/images/products/${images}`} />
                 <Link id="productLink" key={id} to={`/products/${id}`}>
                     <div className="info" id={id}>
-
-                        <StarsRating rating={rate} id={id} accessToken={accessToken}/>
-
+                        <StarsRating rating={rate} color="white" isSelectable={false} />
```
Comment (oleksandrhodzevych) [inline client/src/components/product-list-item/Product-list-item.js]: color="white" - hard code

File: client/src/components/star-rating/Star-rating.js [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
-    );
+  };
+  return (
+    <div className="star-rating" id="starRating" rating={rating}>
+      <StarRatings
+        id="starRating"
+        rating={rating}
+        changeRating={ratingChanging}
+        starSpacing="0"
+        numberOfStars={5}
```
Comment (oleksandrhodzevych) [inline client/src/components/star-rating/Star-rating.js]: numberOfStars={5} - hard code

File: client/src/components/star-rating/Star-rating.js [hunk 5]
```diff
@@ -1,36 +1,41 @@
...
+        starRatedColor={color}
+        starDimension="25px"
```
Comment (oleksandrhodzevych) [inline client/src/components/star-rating/Star-rating.js]: starDimension="25px" - hard code

**Review by oleksandrhodzevych — CHANGES_REQUESTED** [review]

File: client/src/components/comment-form/comment-form.js (hunk 1 above)
Comment (bandvov) [reply client/src/components/comment-form/comment-form.js]: fixed

Comment (bandvov) [reply client/src/components/comment-form/comment-form.js]: fixed

File: client/src/components/product-details/Product-details.js (hunk 2 above)
Comment (bandvov) [reply client/src/components/product-details/Product-details.js]: fixed

File: client/src/components/product-list-item/Product-list-item.js (hunk 3 above)
Comment (bandvov) [reply client/src/components/product-list-item/Product-list-item.js]: fixed

File: client/src/components/star-rating/Star-rating.js (hunk 4 above)
Comment (bandvov) [reply client/src/components/star-rating/Star-rating.js]: fixed

File: client/src/components/star-rating/Star-rating.js (hunk 5 above)
Comment (bandvov) [reply client/src/components/star-rating/Star-rating.js]: fixed

**Pull request merged**